src/NServiceBus.Core/Serialization/SerializationSettingsExtensions.cs

```diff
@@ -0,0 +1,48 @@
+namespace NServiceBus
+{
+    using System;
+    using System.Collections.Generic;
+    using Serialization;
+    using Settings;
+
+    static class SerializationSettingsExtensions
+    {
+        const string AdditionalSerializersSettingsKey = "AdditionalDeserializers";
```

Comment:
Since those extension methods are the only ones that actually work on those settings it makes sense to define it here.